### ISSUE TITLE
Hide mouse cursor (at least in fullscreen focus mode)

### ISSUE THREAD

**Issue by xbonet** [writemonkey/wm3#413]
I'm using WM 3.3.0 in MacOS, Linux and Windows 10, and I've noticed that even though supposedly at least since 2.0 there was a function that "hid" the mouse cursor, I see there's no longer anything like that. In practical terms, I find that often my cursor is in the wrong place at the wrong time, and I have to momentarily pause my writing to move it. And even if it's not blocking the word I'm currently typing, the fact that there's always a mouse cursor on screen--the only element that reminds me this isn't really a typewriter of sorts--doesn't allow me to enjoy myself 100% with WM. Otherwise, WM is as perfect as they come. I've tried so many apps that promise a distraction-free writing experience and WM is the first that has really delivered! So thanks for that! :) As others have mentioned, cloud support would make it perfect... but for now I'd be content with a way of getting rid of my cursor while in focus writing mode.

**Comment by writemonkey** [comment]
I am only testing this on my Mac right now but mouse pointer always disappears when I start typing and then reappears when I move the mouse. This is a system-wide behavior not specific for wm. Is this not true for you?
i.

**Comment by akadormouse** [comment]
Cursor doesn't disappear when typing on Windows 10. Does disappear when I move the mouse.

**Comment by writemonkey** [comment]
When you say cursor, you mean mouse pointer? How can you use the mouse if pointer disappears when you move the mouse?
i.

**Comment by akadormouse** [comment]
Sorry. If I click outside the text area, the cursor disappears. But it doesn't disappear when I type.

**Comment by xbonet** [comment]
> I am only testing this on my Mac right now but mouse pointer always disappears when I start typing and then reappears when I move the mouse. This is a system-wide behavior not specific for wm. Is this not true for you? i.

Nope. ☹ I have WM 3.3.0 installed in all three systems (all 4 sheets files save to my Dropbox, so I can continue writing in any of them) and I can never get the mouse cursor to disappear while writing. 

IDK, maybe it has to do with my specific version of OS: on my Mac I'm still on Mojave; on my PC it's Win 10, as I mentioned; and on the Linux machine I'm running Linux Lite.

**Comment by writemonkey** [comment]
Can someone provide me with a short screencast that clearly shows the issue with the mouse pointer ...
i.

**Comment by xbonet** [comment]
> Can someone provide me with a short screencast that clearly shows the issue with the mouse pointer ... i.

Hi @writemonkey! Sorry for the delay, was swamped at work. I've done a screen recording for you of how the pointer (either in arrow or text caret mode) never disappears, no matter the writing mode, or even while I'm writing. Hope it shows you what you needed to see:

[Screen Recording](https://drive.google.com/file/d/1ievFAiYrNWi1rlTfKAEe-z--m2RmdVVj/view?usp=share_link)

It's not the end of the world, of course. But when I go into No-Distractions mode I like to set my mouse aside and only use my keyboard, as if it were a typewriter. Often, because I'm expecting the pointer to just disappear, I don't move it out of the way, so at some point the text gets to where the pointer is, and the pointer covers the text... so this really distracts me, as I need to then go for my mouse to push it out of the way. And just in general: the fact that the pointer is always on screen just constantly reminds me this is a computer, not a typewriter, so the internet and email and etc is just behind the full screen WM interface!

As I mentioned, this is happening on 3 different machines and OS's where I'm using WM3: MacOS (Mojave), Windows 10 Pro, and Linux Lite.

**Comment by writemonkey** [comment]
I have written a simple plugin that aims to resolve the cursor issue. However, since I do not encounter the cursor problem on my devices, I am unable to test it thoroughly. Please let me know me if this helps at your end.  Download the plugin using the link below.​

iztok​​

https://dropover.cloud/b65c5d
